packages/duration/src/lib/constants.ts

```diff
@@ -1,13 +1,16 @@
 import type { DurationFormatAssetsTime, DurationFormatSeparators } from './DurationFormatter';
 
 export enum Time {
+	Nanosecond = 1 / 1_000_000,
+	Microsecond = 1 / 1000,
 	Millisecond = 1,
 	Second = 1000,
-	Minute = 1000 * 60,
-	Hour = 1000 * 60 * 60,
-	Day = 1000 * 60 * 60 * 24,
-	Month = 1000 * 60 * 60 * 24 * (365 / 12),
-	Year = 1000 * 60 * 60 * 24 * 365
+	Minute = Second * 60,
+	Hour = Minute * 60,
+	Day = Hour * 24,
+	Week = Day * 7,
+	Month = Day * (365 / 12),
+	Year = Day * 365
 }
```

Comment:
This enum also exists in /cron and /timestamp, update it there as well. Do so in a separate PR and separate commits so they it can rebased & merged and each commit only touches 1 package.